### PR TITLE
XW-604 | fix SASS vars for Theme Designer

### PR DESCRIPTION
**Pull request by Warkot**

### CODE REVIEW

File: includes/wikia/services/sass/Compiler/LibSassCompiler.class.php [hunk 1]
```diff
@@ -113,6 +113,7 @@ public static function encodeSassMap( Array $map ) {
 
 	public function quoteIfNeeded( $item, $key ) {
 		if ( !is_numeric( $item ) && strpos( $key, 'color' ) === false ) {
+			$item = addslashes( $item );
```
Comment (hakubo) [inline includes/wikia/services/sass/Compiler/LibSassCompiler.class.php]: so we don't have to wrap it with ""?

**Comment by vforge** [comment]
LGTM, please add this to unittests

**Comment by vforge** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Pull request merged**